SPECS-EXTENDED/openblas/openblas.spec

```diff
@@ -34,6 +34,8 @@ Patch3:         openblas-0.3.7-tests.patch
 # Fix C++ compatibility (BZ #1820131)
 Patch4:         https://github.com/xianyi/OpenBLAS/commit/ee2e758278b5d82b7242f505ea694f082ef65879.patch
 
+Patch5:         No-Fortran-Build.patch
+
 BuildRequires:  gcc
 BuildRequires:  gcc-c++
 BuildRequires:  gcc-gfortran
```

Comment:
Considering the change, can we also remove this dependency?

```suggestion
```

Reply:
I actually kept it and add a comment about why patch 5 is needed (fortran build correctly in a non-containerized env)